src/app_engine/compute_page.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+
+"""Compute page for handling tasks related to compute engine."""
+
+import logging
+
+import webapp2
+
+import apiauth
+import constants
+from google.appengine.api import app_identity
+from google.appengine.api import taskqueue
+
+
+class ComputePage(webapp2.RequestHandler):
+  """Page to handle requests against GCE."""
+
+  compute_service = None
+
+  def __init__(self, request, response):
+    self.initialize(request, response)
+    if ComputePage.compute_service is None:
+      ComputePage.compute_service = self._build_compute_service()
+    if ComputePage.compute_service is None:
+      logging.warning('Unable to create Compute service object.')
+
+  def _build_compute_service(self):
+    return apiauth.build(scope=constants.COMPUTE_URL,
+                         serviceName='compute',
+                         version='v1')
+
+  def _enqueue_start(self):
+    taskqueue.add(url='/compute/start',
```

Comment:
Can we add some authentication info so that an external user cannot do it, but only GAE can issue it? It makes me feel uneasy to allow anyone to manipulate the VM instance.